devtools/server/actors/replay/connection.js

```diff
@@ -533,6 +527,117 @@ class Recording extends EventEmitter {
   }
 }
 
+function checkShouldValidateUrl() {
```

Comment:
Why not make this an async function and use `await` instead of `.then`?

Reply:
I still mentally default to using `.then()` and missed this one was still here after converting the other to async/await.